src/test/kfs_rest_test.cpp

```diff
@@ -255,6 +255,7 @@ TEST_F(HttpRestApiHandlerTest, binaryInputsBYTES) {
     ASSERT_EQ(grpc_request.inputs()[0].shape()[0], 1);
     ASSERT_EQ(grpc_request.inputs()[0].shape()[1], 4);
     ASSERT_EQ(grpc_request.inputs()[0].contents().bytes_contents()[0], binaryData);
+    ASSERT_EQ(grpc_request.inputs()[0].contents().bytes_contents_size(), 1);
```

Comment:
What I meant was that:
```
 ASSERT_EQ(grpc_request.inputs_size(), 1);
```

The line 258 should be here as well, I did not catch it at first sight.
Even though the line will not help in case of missing element 0 in line 257.
We should always check the container size first before accessing the elements, otherwise tests will crash instead of report valuable info.